plugins/ServerFeatures.ts

```
@@ -0,0 +1,572 @@
+
+const cwd = process.cwd();
+
+function isFunction(functionToCheck: any) {
+  return typeof functionToCheck == "function";
+}
+
+/**
+ * this will set this.ssrElements & this.revalidates and make the build out from
+ * a child process to avoid some error while building multiple time from the main process.
+ */
+function ServerActionToClient(func: Function, ModulePath: string): string {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)